### ISSUE TITLE
Error in UPD on single sample

### ISSUE THREAD

**Issue by fa2k** [nf-core/raredisease#642]
### Description of the bug

I run a singleton sample for testing (but it's also a real use case). The pipeline crashes in the process `NFCORE_RAREDISEASE:RAREDISEASE:ANNOTATE_GENOME_SNVS:UPD_REGIONS`.

It seems to put the ID of the parents, which is 0, because they are not provided.

### Command used and terminal output

```
ERROR ~ Error executing process > 'NFCORE_RAREDISEASE:RAREDISEASE:ANNOTATE_GENOME_SNVS:UPD_REGIONS (NA12878)'

Caused by:
  Process `NFCORE_RAREDISEASE:RAREDISEASE:ANNOTATE_GENOME_SNVS:UPD_REGIONS (NA12878)` terminated with an error exit status (1)


Command executed:

  upd \
      --vcf NA12878_rhocall_filter_cadd_vep.vcf.gz \
      --af-tag GNOMADAF --proband NA12878 --mother 0 --father 0 regions --min-size 5 --min-sites 1 \
      | sort -k 1,1 -k 2,2n >NA12878_upd_regions.bed
  
  cat <<-END_VERSIONS > versions.yml
  "NFCORE_RAREDISEASE:RAREDISEASE:ANNOTATE_GENOME_SNVS:UPD_REGIONS":
      upd: $( upd --version 2>&1 | sed 's/upd, version //' )
  END_VERSIONS

Command exit status:
  1

Command output:
  (empty)

Command error:
  INFO:    Environment variable SINGULARITYENV_NXF_TASK_WORKDIR is set, but APPTAINERENV_NXF_TASK_WORKDIR is preferred
  INFO:    Environment variable SINGULARITYENV_NXF_DEBUG is set, but APPTAINERENV_NXF_DEBUG is preferred
  2024-10-29 19:50:30 nepe upd.cli[44] INFO Running upd version 0.1
  2024-10-29 19:50:30 nepe upd.vcf_tools[44] INFO NA12878_rhocall_filter_cadd_vep.vcf.gz is zipped
  2024-10-29 19:50:30 nepe upd.cli[44] WARNING At least one of the given sample IDs do not exist in the VCF header
  Aborted!

Work dir:
  /home/fa2k/local/na12878_med_pipeline/work/11/fb272906f67d525b2cf4cc2c2248b5

Container:
  /data/nobackup/nsc/raredisease-configs/singularity/depot.galaxyproject.org-singularity-upd-0.1.1--pyhdfd78af_0.img

Tip: you can try to figure out what's wrong by changing to the process work dir and showing the script file named `.command.sh`

 -- Check '.nextflow.log' file for details
ERROR ~ Pipeline failed. Please refer to troubleshooting docs: https://nf-co.re/docs/usage/troubleshooting

 -- Check '.nextflow.log' file for details
```

### Relevant files

Sample sheet

```
sample,fastq_1,fastq_2,sex,phenotype,case_id,paternal_id,maternal_id,lane
NA12878,/data/nobackup/nsc/Testdata/subsample_600mreads_1.fastq.gz,/data/nobackup/nsc/Testdata/subsample_600mreads_2.fastq.gz,2,2,NA12878,0,0,1
```

### System information

**dev branch**

**Comment by jemten** [comment]
Thanks for submitting @fa2k. Looking into it!

**Comment by jemten** [comment]
I ran some tests with PR #643 and I think it should resolve your issue. You think you could try that branch?

**Comment by fa2k** [comment]
@jemten It works well now, with this patch. Thanks for the fix :) The whole pipeline completes successfully now.

**Comment by ramprasadn** [comment]
Fixed in #643
